### PR TITLE
Mvn version and dependency fixes for merging to develop branch

### PR DESCRIPTION
**Pull request by onurd86**
Pull request for merging to develop branch.

### CODE REVIEW

File: jdk_17_maven/cs/rest/familie-ba-sak/pom.xml [hunk 1]
```diff
@@ -392,6 +392,10 @@
             <id>github</id>
             <url>https://maven.pkg.github.com/navikt/familie-felles</url>
         </repository>
+        <repository>
+            <id>familie</id>
+            <url>https://github-package-registry-mirror.gc.nav.no/cached/maven-release</url>
+        </repository>
```
Comment (arcuri82) [inline jdk_17_maven/cs/rest/familie-ba-sak/pom.xml:398]: why modifying this?

File: jdk_8_maven/cs/rest-gui/ocvn/pom.xml [hunk 2]
```diff
@@ -84,6 +84,11 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>maven.restlet.org</id>
+            <name>maven.restlet.org</name>
+            <url>https://maven.restlet.talend.com</url>
+        </repository>
```
Comment (arcuri82) [inline jdk_8_maven/cs/rest-gui/ocvn/pom.xml:91]: why modifying this?

File: scripts/dist.py [hunk 3]
```diff
@@ -29,12 +29,24 @@ def checkMavenVersion():
...
+        version_res = run([mvn_path, "-v"], capture_output=True, text=True)
+        result_out = version_res.stdout.strip()
+        match = re.search(MAVEN_VERSION_REGEX, result_out)
+
+        if match == None:
+            print("\nVersion could not be retrieved with mvn -v command")
+
+            #        might happen depending on installation path... instead of crashing immediately,
+            #        we try to build, as it might be the correct version... if not, it will fail anyway
+            #  TODO ideally, should rather use "mvn -v" to determine the version number...
```
Comment (arcuri82) [inline scripts/dist.py:45]: these last 2 comment lines are now no longer relevant

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: jdk_8_maven/cs/rest-gui/ocvn/pom.xml (hunk 2 above)
Comment (onurd86) [reply jdk_8_maven/cs/rest-gui/ocvn/pom.xml:91]: Hi Andrea,

Thanks for your comment. The mvn command failed to fetch ocvn in my machine (MacOS). After searching online, someone suggested that repository so I added that and it worked.

File: jdk_17_maven/cs/rest/familie-ba-sak/pom.xml (hunk 1 above)
Comment (onurd86) [reply jdk_17_maven/cs/rest/familie-ba-sak/pom.xml:398]: Hi Andrea,

Thanks for your comment. The mvn command failed to fetch familie-ba-sak in my machine (MacOS). I saw the repository in the link https://github.com/navikt/github-package-registry-mirror. It worked in my case after adding this repository.

File: scripts/dist.py (hunk 3 above)
Comment (onurd86) [reply scripts/dist.py]: Hi Andrea,

Thanks for your comment. I removed the last two lines.

File: jdk_17_maven/cs/rest/familie-ba-sak/pom.xml (hunk 1 above)
Comment (arcuri82) [reply jdk_17_maven/cs/rest/familie-ba-sak/pom.xml:398]: @onurd86 was this before or after fixing your `settings.xml` file?

Comment (onurd86) [reply jdk_17_maven/cs/rest/familie-ba-sak/pom.xml:398]: @arcuri82 This was after fixing settings.xml file.

File: jdk_8_maven/cs/rest-gui/ocvn/pom.xml (hunk 2 above)
Comment (arcuri82) [reply jdk_8_maven/cs/rest-gui/ocvn/pom.xml:91]: @onurd86  we cannot have dependencies in unreliable external repositories that is not Maven Central nor GitHub. which library gave problems? note that in OCVN we had a similar issue in the past. This was solved with local jar, manually added. see `jdk_8_maven/cs/rest-gui/ocvn/persistence/lib/dozer-hibernate-model-0.3.5-06032016.jar` and the use of 
```
 <dependency>
            <groupId>nl.dries.wicket.hibernate</groupId>
            <artifactId>dozer-hibernate-model</artifactId>
            <version>${dozer.hibernate.model.version}</version>
            <scope>system</scope>
            <systemPath>${project.basedir}/lib/dozer-hibernate-model-0.3.5-06032016.jar</systemPath>
        </dependency>
```
in `jdk_8_maven/cs/rest-gui/ocvn/persistence/pom.xml`

Comment (onurd86) [reply jdk_8_maven/cs/rest-gui/ocvn/pom.xml:91]: Hi Andrea,

In that case, I can remove that one.

Comment (onurd86) [reply jdk_8_maven/cs/rest-gui/ocvn/pom.xml:91]: I saw that the jar is there but it did not work for my case when I removed the repository.

**Pull request closed without merging**